### DISCUSSION TITLE
Auto tangle

### DISCUSSION THREAD

**Discussion by refaelsh** [nvim-neorg/neorg#1240] [Q&A] [answered]
I was wandering, is it possible to auto tangle when ever the file is saved?
I know I can do it with Neovim itself using auto commands in `init.lua`, but I was wondering if it is possible to do it with this plugin's functionality without Neovim's `init.lua`.

**Comment by max397574** [comment, marked as answer]
no it isn't
I'd just write an autocmd `vim.api.nvim_create_autocmd("BufWritePre",{callback=function()vim.cmd("Neorg tangle current-file") end})`

**Reply by refaelsh** [reply]
Thank you. I will use this `autocmd` of yours.
